src/App.js

```diff
@@ -4,9 +4,23 @@ import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export class App extends Component {
-  state = {};
+  state = {
+    query: '',
+  };
+
+  handleQueryChange = (event) => {
+    this.setState({ query: event.target.value });
```

Comment:
```suggestion
    this.setState({ query: event.target.value.toLowerCase() });
```
Это у тебя контролируемый ввод, это позволяет сразу привести к нижнему регистру

Reply:
good advice

Reply:
Круто, це дуже полегшить роботу, дякую!